src/Components/Accordion.js

```diff
@@ -0,0 +1,61 @@
+import React,{useState} from 'react'
+import arrow from '../images/icon-arrow-down.svg';
+import styled,{css} from 'styled-components'
+const Wrapper = styled.div`
+    border-bottom: 1px solid hsl(240, 5%, 91%);
+    padding: 20px 0;
+`;
+const Arrow = styled.img`
+
+    ${props => !props.arrowClose && css`
+        transform: rotateX(180deg);
+    `
+
+    }
+`
+const Answer = styled.div`
+    margin-top: 20px;
+    ${props => props.hide && css`
+        display:none
+    `}
+`
+const QuestionText = styled.div`
+    &:hover{
+            color: hsl(14, 88%, 65%);
+        }
+    ${props => props.bold && css`
+            color: black;
+            font-weight: bold;
+        `}
+
+`
+const QuestionWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+    &:hover{
+        cursor: pointer;
+    }
+`
+const Accordion = (props) => {
```

Comment:
I'd suggest `export default function Accordion(props){}` but it's a matter of personal preference and project code style